packages/flutter/test/material/selection_area_test.dart

```diff
@@ -155,6 +155,7 @@ void main() {
 
     // Backwards selection.
     await gesture.down(textOffsetToPosition(paragraph, 3));
+    await tester.pumpAndSettle();
```

Comment:
Why is this needed now? (and in a few other places below)

Reply:
This is needed because of the switch to `PanGestureRecognizer` -> `TapAndPanGestureRecognizer`. `PanGestureRecognizer` calls `onDown` earlier than `TapAndPanGestureRecognizer`. `PanGestureRecognizer` calls it in `addAllowedPointer`, and `TapAndPanGestureRecognizer` waits for the recognizer to declare victory.

Reply:
Thanks for the explanation!